manifests/charts/gateways/istio-ingress/values.yaml

```diff
@@ -162,6 +162,9 @@ defaults:
     # Default tag for Istio images.
     tag: latest
 
+    # Labels to apply to all resources
```

Comment:
Do these need to be defined in `values.yaml`, or can i just omit

Reply:
If it's settable at the chart level, it should be defined in `values.yaml` at the chart level.

Otherwise there's no obvious way for anyone to know they can set it.